code/modules/power/groundmap_geothermal.dm

```diff
@@ -357,7 +352,239 @@ GLOBAL_VAR_INIT(generators_on_ground, 0)
 	name = "\improper Reinforced Reactor Turbine"
 	is_corruptible = FALSE
 
-#undef GEOTHERMAL_NO_DAMAGE
-#undef GEOTHERMAL_LIGHT_DAMAGE
-#undef GEOTHERMAL_MEDIUM_DAMAGE
-#undef GEOTHERMAL_HEAVY_DAMAGE
+
+/*	Thermo-bluespace generator -- the main, big boy generator that engineers need to be concerned with
+	While active -- Generates a shitton of power and overclocks disk generation speed
+	Xenos can disable it after a short windup, which if the generator is past 50%, triggers a massive explosion */
```

Comment:
see dev gen pins for multiline autodocs:

/**
 * This is a item
 *
 * it does things and supports markup:
 * * like so
 */